### DISCUSSION TITLE
How can I add PDF mini TOC links to nested topics?

### DISCUSSION THREAD

**Discussion by stefan-jung** [dita-ot/dita-ot#4107] [Q&A] [answered]
Assuming I have nested topics, like so...

```xml
<topicref href="foo.dita">
  <topicref href="bar.dita"/>
</topicref>
```

The PDF mini TOC has entries to both topics. But when nesting the topics, I do not get a mini TOC entry for `bar.dita`. (How) can I change that?

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE topic PUBLIC "-//OASIS//DTD DITA Topic//EN" "topic.dtd">
<topic id="foo">
    <title>foo</title>
    <topic id="bar">
        <title>bar</title>
    </topic>
</topic>
```

**Comment by raducoravu** [comment, marked as answer]
How about if you refer to both topic and inner topic in the DITA Map? Like:

    <topicref href="foo.dita">
      <topicref href="foo.dita#bar"/>
    </topicref>

**Reply by stefan-jung** [reply]
Hello @raducoravu, unfortunately this does not help, but I assume your solution is correct. I have figured out our problem is not the topic nesting, it is a general "problem/behavior" of the mini TOC function. I have raised another question: https://github.com/dita-ot/dita-ot/discussions/4116 Therefore, I'll mark your answer as correct, although I could not verify this so far.
